doc/source/parameters/dem/simulation_control.rst

```diff
@@ -0,0 +1,30 @@
+Simulation Control
+-------------------
+This subsection contains the general information of the simulation, including ``time step``, ``end time``, ``output path``, ``log frequency``, and ``output frequency``. It is the most commonly modified section for a simulation. ``time step`` in DEM simulations is generally in the range of 1e-7 to 1e-5 seconds. With this small ``time step``, the DEM simulation can capture a single collision in several iterations, which leads to the high accuracy of the simulation. 
```

Comment:
These parameters were already explained in the other simulation control right. So you could maybe instead just give details on the specificity of the simulation control in the case of DEM.